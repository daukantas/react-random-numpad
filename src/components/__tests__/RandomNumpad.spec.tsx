import * as React from 'react';

import { shallow } from 'enzyme';
import RandomNumPad from '../RandomNumPad';

describe('RandomNumPad Component Suite', () => {
  const getComponent = () => shallow(<RandomNumPad />);

  describe('FocusOnScroll Component', () => {
    test('should render with defaults', () => {
      expect(getComponent()).toMatchSnapshot();
    });
  });
});
