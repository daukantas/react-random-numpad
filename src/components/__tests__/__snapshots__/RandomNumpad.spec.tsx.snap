// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RandomNumPad Component Suite FocusOnScroll Component should render with defaults 1`] = `
<div>
  This is a number pad
</div>
`;
